build.sh

```diff
@@ -71,27 +71,31 @@ fi
 dep_install(){
 	  if [ "$distribution" = "CentOS" ] || [ "$distribution" = "Red\ Hat" ] || [ "$distribution" = "Oracle" ]; then
 	  sudo yum install -y libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel alsa-lib-devel libXi-devel > /dev/null 2>&1
-		
+
     elif [ "$distribution" = "Fedora" ]; then
 	  sudo dnf install -y libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel alsa-lib-devel libXi-devel > /dev/null 2>&1
-		
+
     elif [ "$distribution" = "Debian" ] || [ "$distribution" = "Ubuntu" ] || [ "$distribution" = "Deepin" ]; then
 	  sudo apt-get install -y libxcursor-dev libxrandr-dev libxinerama-dev libxi-dev libgl1-mesa-dev libsdl2-dev libasound2-dev > /dev/null 2>&1
-		
+
     elif [ "$distribution" = "Gentoo" ]; then
     sudo emerge --ask n libXcursor libXrandr libXinerama libXi libGLw libglvnd libsdl2 alsa-lib > /dev/null 2>&1
-		
+
     elif [ "$distribution" = "Manjaro" ] || [ "$distribution" = "Arch\ Linux" ]; then
 
-    if [ -z "$mesa_detect_arch" ]; then
-		sudo pacman -S libxcursor libxrandr libxinerama libxi mesa libglvnd sdl2 sdl2_mixer sdl2_net alsa-lib --noconfirm > /dev/null 2>&1
-    else
-    sudo pacman -S libxcursor libxrandr libxinerama libxi libglvnd sdl2 sdl2_mixer sdl2_net alsa-lib --noconfirm > /dev/null 2>&1
-    fi
+      if [ -z "$mesa_detect_arch" ]; then
+  		sudo pacman -S libxcursor libxrandr libxinerama libxi mesa libglvnd sdl2 sdl2_mixer sdl2_net alsa-lib --noconfirm > /dev/null 2>&1
+      else
+      sudo pacman -S libxcursor libxrandr libxinerama libxi libglvnd sdl2 sdl2_mixer sdl2_net alsa-lib --noconfirm > /dev/null 2>&1
+      fi
 
 	  elif [ "$distribution" = "OpenSUSE" ] || [ "$distribution" = "SUSE" ]; then
 		sudo zypper install -y libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel Mesa-libGL-devel alsa-lib-devel libXi-devel > /dev/null 2>&1
 
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+    # are there dependencies required? did I just have all of them already?
+    echo "Mac OS detected, no dependency installation necessary..."
```

Comment:
Should this even be here? I don't particularly like it, but I do want to give a future contributor a clear place to add OS X dependencies without having to figure out *how* to even detect OS X.

Reply:
For MacOS if Golang is installed, it will simply work. I haven't had to install third party stuff that I know of.